countdown/countdown.js

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env node
+
+function getValue() {
+    var timerValue = Number(document.getElementById("input").value);
+    return timerValue;
+    // var myInterval = setInterval(update, 1000, timerValue);
+}
+
+function startInterval() {
+    var countdownInterval = setInterval(countDown, 1000, getValue());
```

Comment:
I'm betting you'll run into an issue with clearing the interval like this, perhaps a single function to manage interval states might be better?